app/assets/javascripts/think_feel_do_dashboard/participants/_form.js

```diff
@@ -8,14 +8,19 @@
     return Math.floor(Math.random() * (max - min)) + min;
   }
 
+  function capitalizeFirstLetter(string) {
+    return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
+  }
+
   ParticipantForm.generatePassword = function() {
     var password = [];
 
     for (var i = 0; i < PASSPHRASE_LENGTH; i++) {
-      password.push(words()[getRandomInt(0, words().length)]);
+      var passwordFragment = words()[getRandomInt(0, words().length)];
+      password.push(capitalizeFirstLetter(passwordFragment));
     }
 
-    this.password = password.join(' ');
+    this.password = password.join(getRandomInt(0,9));
```

Comment:
This join method is adding a randomized integer in between each of the randomized words in the password array.